src/operator/concat.ts

```diff
@@ -10,9 +10,9 @@ import {MergeAllOperator} from './mergeAll-support';
  * on to the next.
  * @params {...Observable} the observables to concatenate
  * @params {Scheduler} [scheduler] an optional scheduler to schedule each observable subscription on.
- * @returns {Observable} All values of each passed observable merged into a single observable, in order, in serial fashion.
+* @returns {Observable} All values of each passed observable merged into a single observable, in order, in serial fashion.
  */
-export function concat<R>(...observables: (Observable<any> | Scheduler)[]): Observable<R> {
+export function concat<T, R>(...observables: Array<Observable<any> | Scheduler>): Observable<R> {
```

Comment:
This line https://github.com/ReactiveX/RxJS/pull/1077/files#diff-261ac0fcd1adcd646e3747b9b8bf73f0R15 does 
`concat<T, R>(...observables: Array<Observable<any> | Scheduler>): Observable<T>`

returns `Observable<T>` while this returns `Observable<R>`, is there any differences?